.devenv.flake.nix

```diff
@@ -0,0 +1,80 @@
+{
```

Comment:
This file shouldn't be committed. It should be catched by the gitignore actually